src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -600,6 +601,190 @@ public IActionResult GetHistory([FromQuery] WalletHistoryRequest request)
             }
         }
 
+        /// <summary>
+        /// Gets the full history of a wallet. This includes the transactions held by the entire wallet
+        /// or a single account if one is specified. 
+        /// </summary>
+        /// <param name="request">An object containing the parameters used to retrieve a wallet's full history.</param>
+        /// <returns>A JSON object containing the full wallet history.</returns>
+        [Route("fullhistory")]
+        [HttpGet]
+           public IActionResult GetFullHistory([FromQuery] WalletHistoryRequest request)
```

Comment:
Awesome! A thought...

A wallet with a lot of transactions would have issues sending a large JSON request back, and fail.

We should probably add paging to GetHistory instead.